### ISSUE TITLE
Question about error handling in services 

### ISSUE THREAD

**Issue by spetca** [angelguzmaning/ts-redux-react-realworld-example-app#6]
I have a question about the conduit.ts file in /services. Taking updateArticle and getArticle as an example, some of the functions in this file omit try/catch's. Is it to obfuscate back end failure from the front end - i.e. the only errors we care about are errors that were caused by sending data from the front end? I see that the only places that have try/catch's are where axios is receiving a second argument, for the most part. There are 1 or 2 exceptions to that pattern I’m seeing. 

wondering about the reasoning for omitting try catches on some of these? or if you have any resources to read more about this?

 In short, why wouldnt you always use a Result type and thus a try/catch scheme, at least for the sake of consistency within this file? 

thanks!

**Comment by angelguzmaning** [comment]
I am assuming that the API is working properly all the time and only try/catching when there is an expected error. Returning `Result` and catching all requests is definitely the better way and that's why I love Elm as it forces you to do that. So the only reason I didn't follow those guidelines was that I considered it to be out of scope for the initial implementation of this project, but it is something that could be added eventually.
